src/plugins/intel_cpu/src/nodes/executors/acl/acl_convert.hpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "nodes/executors/convert.hpp"
+#include "utils/debug_capabilities.h"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class ACLConvertExecutor : public ConvertExecutor {
+public:
+    explicit ACLConvertExecutor(const ExecutorContext::CPtr context);
+    bool init(const ConvertParams& convertParams,
+              const std::vector<MemoryDescPtr>& srcDescs,
+              const std::vector<MemoryDescPtr>& dstDescs,
+              const dnnl::primitive_attr &attr) override;
+    void exec(const std::vector<MemoryCPtr>& src, const std::vector<MemoryPtr>& dst) override;
+    impl_desc_type getImplType() const override { return implDescType; };
+    ~ACLConvertExecutor() override = default;
```

Comment:
This declaration doesn't make much sense.

Reply:
removed